djangocms_picture/migrations/0003_migrate_to_filer.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import filer.fields.image
+
+
+def migrate_to_filer(apps, schema_editor):
```

Comment:
please split migrations by logic, field changes are one type of operation and data migration is another